### ISSUE TITLE
 I am using datasets of size (480,52) as my training inputs. i am trying to reshape it before adding it to Conv2D layer.. but my reshape syntax is not working .. can some one say me a method how can i reshape it?

### ISSUE THREAD

**Issue by MonicaKrishnamoorthy** [keras-team/keras#10725]
Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) and ask there instead of filing a GitHub issue.

Thank you!

- [ ] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/keras-team/keras.git --upgrade --no-deps

- [ ] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by MikeShlis** [comment]
Conv2D requires a channel axis, so if by (480, 52) you mean each data sample is of that size, then you could use something like this 

```
inpt = Input(shape=(480, 52))
inpt_reshaped = Reshape((480,52,1))
```

or just reshape the samples before pushing them to the model

**Comment by MonicaKrishnamoorthy** [comment]
i tried with this but still its not working.. actually the datasets works for neural network only for  convnets i have a problem in reshaping. i tried to reshape it but shows the array cannot be reshaped

**Comment by MikeShlis** [comment]
Can you show a snippet?
